ArcGISRuntimeSDKQt_CppSamples/Maps/ChangeBasemap/main.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright 2015 Esri.
+// Copyright 2021 Esri.
```

Comment:
FYI for future - copyright does not need to get updated. It just needs to be current to the year it was created. It's fine to update it though - no issues, just not required